docs/import/manage_disaster.js

```diff
@@ -512,6 +530,104 @@ function createAssetDropdown(
   return select;
 }
 
+/**
+ * Sets off a column verification check and data write.
+ * @param {Object} event selector change event
+ * @param {Array<string>} propertyPath List of attributes to follow to get
+ *     value.
+ * @param {Array<string>} expectedColumns
+ * @param {string} type
+ * @param {string} state
+ * @return {Promise<void>} see {@link verifyAsset}
+ */
+function onNonDamageAssetSelect(
+    event, propertyPath, expectedColumns, type, state) {
+  const newAsset = $(event.target).val();
+  handleAssetDataChange(newAsset, propertyPath);
+  return verifyAsset(newAsset, type, state, expectedColumns);
+}
+
+// Map of asset picker (represented by a string '<type>-<state>' e.g.
+// 'poverty-WA') to the most recent value of the picker. This helps us ensure
+// that we're only updating the status box for the most recently selected value.
+const lastSelectedAsset = new Map();
+
+/**
+ * Verifies an asset exists and has the expected columns.
+ * @param {string} asset
+ * @param {string} type values from the first index of each entry in {@code
+ *     scoreAssetTypes}
+ * @param {string} state e.g. 'WA'
+ * @param {Array<string>} expectedColumns
+ * @return {Promise<void>} returns null if column checking wasn't needed.
+ * Otherwise returns a promise that resolves when column checking is finished
+ * and select border color is updated.
+ */
+function verifyAsset(asset, type, state, expectedColumns) {
+  // TODO: disable kick off button until all green?
+  const tdId = type + '-' + state;
+  const select = $('#select-' + assetSelectionRowPrefix + type + '-' + state);
+  lastSelectedAsset.set(tdId, asset);
+  const assetMissingErrorFunction = (err) => {
+    if (err.includes('\'' + asset + '\' not found.')) {
+      updateColorAndHover(select, 'red', 'Error! asset could not be found.');
+    } else {
+      console.error(err);
+      updateColorAndHover(select, 'red', 'Unknown error.');
```

Comment:
Any reason not to show the error in the hover message? Like, "Unknown error: ..."

Reply:
Ah, sure.